recipes-support/skl/skl_git.bb

```diff
@@ -7,9 +7,8 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=4641e94ec96f98fabc56ff9cc48be14b"
 
 DEPENDS = "util-linux-native"
 
-SRC_URI = "git://github.com/TrenchBoot/secure-kernel-loader.git;protocol=https;branch=${BRANCH};name=skl"
-BRANCH = "shrink_measured_size"
-SRCREV = "18f535843b22e784dbbc8e1ec11d8dc099b3d0fe"
+SRC_URI = "git://github.com/TrenchBoot/secure-kernel-loader.git;protocol=https;nobranch=1;tag=${TAG};name=skl"
+TAG = "v0.4.0"
```

Comment:
`This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.`

Reply:
It was probably rebased, this commit has identical commit message: https://github.com/TrenchBoot/secure-kernel-loader/commit/794fe07a4e85a19844eb0f5a76f1337976dbb503 and the same amount of changed lines.

Reply:
It was, and it will be in the future.
We were supposed to use this tag right now, and so we have.
This has been already confirmed by @krystian-hebel .

We can (and will) change this revision in the future as needed.